hera_mc/cm_partconnect.py

```diff
@@ -274,69 +274,72 @@ def get_part_revisions(hpn, session=None):
         revisions[parts_rec.hpn_rev]['hpn'] = hpn  # Just carry this along
         revisions[parts_rec.hpn_rev]['started'] = parts_rec.start_date
         revisions[parts_rec.hpn_rev]['ended'] = parts_rec.stop_date
-    if close_session_when_done:
+    if close_session_when_done:  # pragma: no cover
         session.close()
     return revisions
 
 
-class Dubitable(MCDeclarativeBase):
+class AprioriAntenna(MCDeclarativeBase):
     """
-    A table to track the dubitable antennas.
+    Table for a priori antenna status.
 
-    start_gpstime:  start time of list
-    stop_gpstime:  stop time of list
-    ant_list:  list of dubitable antennas.
-               It is a comma-separated list of antenna integers.  E.g. 1,4,5,23,51
-               (No spaces betweeen).
+    antenna:  antenna designation, e.g. HH123
+    start_gpstime: start time for antenna status
+    status:  enum of status - 'passed_checks', 'needs_checking', 'known_bad', 'not_connected'
```

Comment:
This docstring should mention `stop_gpstime` now that it's a column in the table again.